pages/admin/[reviewEditId].tsx

```diff
@@ -57,15 +68,16 @@ export default function ReviewPage(props: {
     level: { original: string, current?: string },
     name: { original: string, current?: string },
     date: string,     //date taught student, might be null
-    examBoard: string   //might be null
+    examBoard: string,   //might be null
```

Comment:
Types that could be null should have this in the type definition, e.g.
```ts
examBoard: string | null
```

Reply:
ok, strings can be null anyways though right?

Reply:
I think technically yes, but it's good practice if they're ever possibly going to be null to mark them as such.